pkg/dns/vips_allocator_test.go

```diff
@@ -208,6 +208,7 @@ var _ = Describe("VIP Allocator", func() {
 
 type outboundViewTestCase struct {
```

Comment:
I don't think we can really. Like I mentioned it doesn't actually change anything visible since it just adds copies of the entries added from `Dataplane`s. The change will come in the crossmesh crosszone PR